README.md

```diff
@@ -1,89 +1,494 @@
+
+@dwyl we write code to make useful things for real people. <br />
+<!-- We don't write code for hedge funds and high frequency traders
+to [help rich people get richer](https://youtu.be/_HDFegpX5gI). -->
+We do not expect that _all_ the people
+doing the Apprenticeship to write code for their _career_.
+In fact we expect people to gravitate
+to the area of creativity they most enjoy
+and is most fulfilling/valuable to them.
+In the same way that reading, writing and math
+is required in all schools
+(_an is widely considered to be "basic literacy"_)
+we feel that _knowing_ how to code is _essential_.
+
+<!--
+In the same way that not _all_ people
+who are learning to code
+as part of primary school curriculum
+in schools will write code for a living.
+-->
+
+#### “_Whether you want to uncover the secrets of the universe, or you just want to pursue a career in the 21st century, basic computer programming is an essential skill to learn_.” ~ Stephen Hawking
+
+Along the way you
+_use_ technology to leverage your creativity
+***building*** technology products
 with _great_ UX and code quality!
 
+
+
 ## Who?
 
-Anyone who has not worked as a "developer" before and wants to get started!
+Anyone _**over 18** years old_<sup>1</sup>
+who has
+***either*** not worked in creative tech before
+before and wants to get started,
+***or*** people that have worked in tech
+and want to explore a different path.
+
+The apprenticeship is for self-motivated people
+who are eager to **learn _fast_ and _make_ things**! <br />
+If you are _not already_ motivated
+to learn _independently_,
+please
+[do _not_ apply](https://github.com/nelsonic/nelsonic.github.io/issues/318)
+to ACT
+until you learn to
+[_apply_ yourself](https://english.stackexchange.com/questions/267667/apply-yourself).
+
+
+> <small><sup>1</sup>The reason for **minimum age**
+for ACT is simple:
+we don't want anyone quitting (regular) school,
+[running](https://www.quora.com/Did-people-actually-used-to-run-away-and-join-the-circus-And-do-they-still-do-so-now)
+[away](https://www.wikihow.com/Run-Away-With-the-Circus)
+from their parents home
+and showing up on our doorstep!
+We _encourage_ anyone/everyone
+_under_ age 18 to start their creative tech learning journey
+as soon as possible!
+But _definitely_ "stay in school"
+and learn everything you can,
+because life it's much easier
+if you tick the box of "school" first.
+If you are still in high school
+and feel that your creativity
+is _not_ being rewarded,
+don't worry or lose hope,
+it will be soon enough! </small>
+
 
 ## How?
 
 ### Prerequisites
 
-+ Read (_and "star"_) `this` repo to show you have (_have read_)
++ **Read** (_and "star"_) `this` repo to show you have (_have read_)
 and are interested in the apprenticeship.
-+  Read (_and "star"_) the dwyl contributing guide:
++ **Read** (_and "star"_) https://github.com/dwyl/start-here
+to show you _agree_ with @dwyl's
+[mission](https://github.com/dwyl/start-here/blob/master/mission.md),
```

Comment:
@iteles please make the _suggestion_ in the form:
```suggestion
[mission](https://github.com/dwyl/start-here/blob/master/mission.md#what-is-dwyls-mission),
```
So it's _easy_ to just commit and move on.

Reply:
I _tried_ doing this but the option was greyed out for me yesterday. Thanks for adding 👍

Reply:
@iteles whenever the _interface_ is greyed out:
![image](https://user-images.githubusercontent.com/194400/73068787-0228ac00-3ea4-11ea-9321-f81872c01c55.png)
You can still make suggestions using the following pattern:
```diff
```suggestion
- old text
+ whatever you want to suggest
... ```
```

Reply:
Note that because I _made_ the suggestion, I cannot commit it:
![image](https://user-images.githubusercontent.com/194400/73068940-60558f00-3ea4-11ea-9fb3-57ad5a3a96a2.png)

You need to commit/accept the suggestion I made above. 🙄

Reply:
Already had: https://github.com/dwyl/apprenticeship/pull/23/commits/ee1345f9d5bb6f4cb211abf47296056586a7f169